docs/service-catalog/06-02-usage-kind.md

```diff
@@ -27,6 +27,14 @@ spec:
   labelsPath: spec.template.metadata.labels
 ```
 
+The binding process modifies the target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod, and the resource type defined by the **group**, **kind**, and **version** fields.
```

Comment:
```suggestion
The binding process modifies the target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod, as well as the resource type defined by the **group**, **kind**, and **version** fields.
```
I'd make this change to avoid ambiguity and cut down on `and`'s in this sentence.